src/scss/_colors.scss

```diff
@@ -1 +1,2 @@
 $wolox-blue: #3BB5CC;
+$disabled: #DADADA;
```

Comment:
We could use this link to name the colors.
https://chir.ag/projects/name-that-color